### ISSUE TITLE
[BUG] can't do output split

### ISSUE THREAD

**Issue by qvieth** [luk400/vim-jukit#29]
![Screenshot from 2022-10-23 15-19-02](https://user-images.githubusercontent.com/71810726/197379445-896a645e-72d1-430b-9a56-cc8318591c5a.png)

from this screen do <leader>os goes to this:

![Screenshot from 2022-10-23 15-18-53](https://user-images.githubusercontent.com/71810726/197379451-e32652af-6101-4fd2-8807-c94b1f350006.png)

**Comment by igorlfs** [comment]
It looks like you don't have matplotlib installed, which is a dependency for python

**Comment by luk400** [comment]
are you using tmux? if so, please try specifying `let g:jukit_terminal = "nvimterm"` in your config, does that fix it?

**Comment by qvieth** [comment]
> It looks like you don't have matplotlib installed, which is a dependency for python

installed matplotlib but it only solved the missing matplotlib error, the terminal still appear wrong(its create a new window and override the coding window I was in)
![image](https://user-images.githubusercontent.com/71810726/197579562-a3fa4b6d-6540-4859-a0e2-b93b51040dd1.png)

**Comment by qvieth** [comment]
> are you using tmux? if so, please try specifying `let g:jukit_terminal = "nvimterm"` (or `"vimterm"` when using vim) in your config, does that fix it?
yes I'm using tmux. Added your `let g:jukit_terminal = "nvimterm"`  and it fixed!
![image](https://user-images.githubusercontent.com/71810726/197580049-31a4a2c9-ee98-48f3-ad04-22f74fcc1a23.png

**Comment by luk400** [comment]
> the plugin is so awesome btw!

thank you, glad you like it! :)
